src/ext/VSExtension/wixext/VSHeatExtension.cs

```diff
@@ -11,6 +11,9 @@
 // </summary>
 //-------------------------------------------------------------------------------------------------
 
+using System.Net.Mime;
+using System.Text.RegularExpressions;
+
```

Comment:
Typically, as seen below, a using goes inside the namespace. Also, what is the purpose of System.Net.Mime here?